src/api/orders/types.ts

```diff
@@ -0,0 +1,33 @@
+import {LimitOrderV3Struct} from '../../limit-order'
+import {NetworkEnum} from '../../constants'
+
+export type OrdersApiConfig = {
+    network: NetworkEnum
+    url: string
+}
+
+export type ActiveOrdersRequestParams = {
+    page?: number
+    limit?: number
+}
+
+export type ActiveOrder = {
+    orderHash: string
+    signature: string
+    deadline: string
+    auctionStartDate: string
+    auctionEndDate: string
+    order: LimitOrderV3Struct
+}
+
+export type ActiveOrdersMeta = {
+    totalItems: number
+    currentPage: number
+    itemsPerPage: number
+    totalPages: number
+}
```

Comment:
Call it `PaginationMeta`

Reply:
Extract into separate types in the root folder